test/runtests.jl

```diff
@@ -22,35 +23,43 @@ base = ["test_abstract_vectors.jl",
         "test_composite.jl",
         "test_judiWeights.jl",
         "test_judiWavefield.jl",
-        "test_linear_operators.jl"]
+        "test_linear_operators.jl",
+        "test_physicalparam.jl"]
 
 devito = ["test_linearity.jl",
+          "test_adjoint.jl",
+          "test_all_options.jl",
           "test_jacobian.jl",
           "test_jacobian_extended.jl",
-          "test_adjoint.jl",
           "test_gradient_fwi.jl",
-          "test_all_options.jl",
           "test_gradient_lsrtm.jl"]
+        #   "test_gradient_twri.jl"]
```

Comment:
should we uncomment this?

Reply:
No this fails most of the time. And talking with @grizzuti he's ok commenting it out since the gradient seems fine by itself.